### PR TITLE
Fix makefiles

### PR DESCRIPTION
**Pull request by Frassle**
# Description

Fix the makefiles to run build/test/brew on the Go and NodeJS language plugins correctly.

### CODE REVIEW

**Review by Zaid-Ajaj — APPROVED** [review]
Tested locally, works like a charm 🚀

**Comment by Frassle** [comment]
bors merge

File: build/common.mk [hunk 1]
```diff
@@ -116,6 +118,10 @@ GO_TEST_FLAGS = -count=1 -cover -tags="${GO_TEST_TAGS}" -timeout 1h \
 	-race=${GO_TEST_RACE} \
 	${GO_TEST_OPTIONS}
 GO_TEST_FAST_FLAGS = -short ${GO_TEST_FLAGS}
+
+GO_TEST      = $(PYTHON) $(ROOT_DIR)/../scripts/go-test.py $(GO_TEST_FLAGS)
+GO_TEST_FAST = $(PYTHON) $(ROOT_DIR)/../scripts/go-test.py $(GO_TEST_FAST_FLAGS)
```
Comment (RobbieMcKinstry) [inline build/common.mk:123]: ❤️

**Review by abhinav — COMMENTED** [review]
sdk/nodejs/pulumi-language-nodejs is a separate Go module?
Okay, we also need to update LINT_GOLANG_PKGS in the root Makefile.

**Pull request merged**